src/test/scala/dibl/PlugInDemos.scala

```diff
@@ -0,0 +1,27 @@
+package dibl
+
+import dibl.proto.TilesConfig
+
+object PlugInDemos extends DemoFixture {
+  def main(args: Array[String]): Unit = {
+    // patch(aka swatch) size >= tile size; tile layout is simple G.W-F4
+    val cfg = TilesConfig("patchWidth=4&patchHeight=4&d1=ctc&c1=ctc&b1=ctc&a1=ctc&d2=ctc&c2=ctcllctc&a2=ctcrrctc&d3=ctc&c3=ctc&b3=ctc&a3=ctc&c4=ctc&b4=ctc&a4=ctc&tile=1483,8-48,8314,488-&footsideStitch=ctctt&tileStitch=ctc&headsideStitch=ctctt&shiftColsSW=0&shiftRowsSW=4&shiftColsSE=4&shiftRowsSE=4")
+
+    def node(row: Int, col: Int) = {
+      val sources = cfg.getItemMatrix(row)(col).relativeSources
+      if (sources.length != 2) ""
+      else {
+        val Array((x1, y1), (x2, y2)) = sources
+        s"[$x1,$y1,$row,$col,$x2,$y2]"
+      }
+    }
+
+    val width = cfg.centerMatrixCols
+    val height = cfg.centerMatrixRows
+    val links = for {
+      row <- 0 until height
+      col <- 0 until width
+    } yield node(row, col)
+    println(links.filterNot(_.isEmpty).mkString(s"$width\t$height\n", "\n", ""))
```

Comment:
![image](https://user-images.githubusercontent.com/10553630/102006635-b16fab00-3d22-11eb-883e-d788cd40ce2c.png)
I forgot you work with outgoing pairs and I with incoming. Shouldn't that just have flipped each tile? It seems worse.

      4	4
      [-1,0,0,0,0,1][-1,0,0,1,-1,1][0,-1,0,2,-1,0][0,-1,0,3,0,1][0,-1,1,0,-1,0][-1,0,1,2,-1,1][0,-1,1,3,-1,0][0,-1,2,0,-1,0][0,-1,2,1,0,1][-1,0,2,2,0,1][-1,0,2,3,-1,1][-1,0,3,0,-1,1][0,-1,3,1,-1,0][0,-1,3,2,-1,0]

Reply:
It is even worse with absolute sources 
`s"[${col+x1},${row+y1},$row,$col,${col+x2},${row+y2}]"`
![image](https://user-images.githubusercontent.com/10553630/102006885-b9304f00-3d24-11eb-983a-5fe49439b962.png)

Reply:
Prefixed the first line with "CHECKER", also tried 
`s"[$row,$col,${col+x1},${row+y1},${col+x2},${row+y2}]"`
and
`s"[$row,$col,$x1,$y1,$x2,$y2]"`
nothing but chaos.